Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/PassSystem.h

```diff
@@ -75,6 +75,12 @@ namespace AZ
             const AZ::Name& GetTargetedPassDebuggingName() const override;
             void ConnectEvent(OnReadyLoadTemplatesEvent::Handler& handler) override;
             PassSystemState GetState() const override;
+            SwapChainPass* FindSwapChainPass(AzFramework::NativeWindowHandle windowHandle) const override;
```

Comment:
Is it possible to return const SwapChainPass* ?
I think if not I would drop the const from the method definition.

Reply:
This is just moving an existing line of code to match the organization of the comments, nothing new :)